visualization/dashboard/src/MLIDashboard/Controls/WhatIfTab/WhatIfTab.tsx

```diff
@@ -695,7 +748,67 @@ export class WhatIfTab extends React.PureComponent<IWhatIfTabProps, IWhatIfTabSt
                                 unsortedSeries={this.includedFeatureImportance}
                                 topK={this.state.topK}
                             />
-                            <div className={classNames.featureImportanceLegend}> </div>
+                            <div className={classNames.featureImportanceLegend}>
```

Comment:
it seems like this feature importance legend could be moved into its own component with render function?

Reply:
Definitely an option. At this time, I'll leave it in this component as its not used elsewhere, and moving it would change the state management.